src/components/AFEpage/index.js

```diff
@@ -1,85 +1,199 @@
 import React from "react";
-import "./styles.scss";
+import useStyles from "./styles";
+import { breakpoints } from "../../theme/constant";
+import {
+  Container,
+  Typography,
+  Grid,
+  useMediaQuery,
+  List,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import FiberManualRecordIcon from "@mui/icons-material/FiberManualRecord";
 
 function AFEpage() {
+  const classes = useStyles();
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
```

Comment:
This is used in a number of places so you may just want to import this as a function from utils.js:

```
export function isSmallScreen() {
  return useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
}
```

```suggestion
  const isActive = isSmallScreen();
```

Reply:
![image](https://user-images.githubusercontent.com/84696578/162616878-d0a83141-c11a-4886-985d-c46675fb916d.png)

Reply:
@kartiks26 Hmm...I was able to get it to work on CodeSandbox despite the error.  But I should have changed the name to indicate that it was a [custom hook](https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook) so the compiler didn't produce an error like this.  To do this, all you need to do is make it start with `use` so the following should (hopefully) work:

```
export function useIsSmallScreen() {
  return useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
}
```